api/lib/application/account-recovery/account-recovery-controller.js

```diff
@@ -4,10 +4,16 @@ const userSerializer = require('../../infrastructure/serializers/jsonapi/user-se
 module.exports = {
 
   async sendEmailForAccountRecovery(request, h) {
-    const userId = request.payload.data.attributes['user-id'];
-    const email = request.payload.data.attributes.email;
+    const payload = request.payload.data.attributes;
```

Comment:
**chipotage**
Pourquoi pas une deserializer comme [celui-là](http://github.com/1024pix/pix/blob/dev/api/lib/infrastructure/serializers/jsonapi/student-information-for-account-recovery-serializer.js) ?

Reply:
done